html-lists/index.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+  </head>
+  <body>
+    <h1>HTML Lists</h1>
+    <div>
+      <h2>Ordered List Example</h2>
+      <ol>
+        <li>1. Learn to code</li>
+        <li>2. Build projects</li>
+        <li>3. Get a job</li>
```

Comment:
You don't need the 1, 2, 3. By using an `ol`, `li` will automatically add those numbers for you.